sdk/datamigration/Microsoft.Azure.Management.DataMigration/src/Microsoft.Azure.Management.DataMigration.csproj

```diff
@@ -6,7 +6,7 @@
 	<PropertyGroup>
 		<PackageId>Microsoft.Azure.Management.DataMigration</PackageId>
 		<Description>Provides developers with libraries for the Database Migration Service (DMS) under Azure Resource manager to deploy an instance of DMS. Create, Delete, Read Projects, Files and Tasks and more.</Description>
-		<Version>0.9.0-preview</Version>
+		<Version>0.10.0-preview</Version>
 		<AssemblyName>Microsoft.Azure.Management.DataMigration</AssemblyName>
 		<PackageTags>DataMigration;Data Migration;DataMigration_2018-07-15-preview;</PackageTags>
 		<PackageReleaseNotes>Preview release of the Database Migration Service (DMS) SDK to Create, Delete, Read Projects, Files and Database Migration Tasks. The SDK now supports multiple online scenarios for targets like Azure SQL Managed Instance, Azure PostgreSQL and Azure MySQL</PackageReleaseNotes>
```

Comment:
Update `PackageReleaseNotes` please

Reply:
No features were added or removed. This is just a small bug fix release. This element hasn't been changed since the initial release as I believe our PMs want this exact wording to accompany the package.

Reply:
https://github.com/Azure/azure-rest-api-specs/pull/7379